src/main/java/games/strategy/engine/data/GameData.java

```diff
@@ -515,6 +515,8 @@ public String toString() {
     sb.append(gameName);
     sb.append(", gameVersion=");
     sb.append(gameVersion);
+    sb.append(", loader=");
+    sb.append(loader);
```

Comment:
Not sure where and how the toString() method is used. Can you confirm this doesn't have any negative impacts or compatibility issues?